editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -280,6 +280,8 @@ class CanvasItemEditor : public VBoxContainer {
 		Transform2D xform;
 		float length;
 		uint64_t last_pass;
+
+		BoneList() : length(0.f), last_pass(0) {}
```

Comment:
https://travis-ci.org/godotengine/godot/jobs/436896280#L537

Reply:
It's a default constructor to fix the following cppcheck errors. I initialized length as a free bonus.

[editor/plugins/canvas_item_editor_plugin.cpp:3067]: (error) Uninitialized struct member: b.last_pass
[editor/plugins/canvas_item_editor_plugin.cpp:3081]: (error) Uninitialized struct member: b.last_pass
[editor/plugins/canvas_item_editor_plugin.cpp:3097]: (error) Uninitialized struct member: b.last_pass

I will reformat it to please clang.